README.md

```diff
@@ -1,12 +1,76 @@
-# Webpack Boilerplate
+# Pokemon
 
->  A simple yet powerful webpack boilerplate, which you can later use as a starting point in your projects.
+>  **Pokemon** is a webapp that simulates a pokedex showing the pokemon of a certain type. The user can choose the type of pokemon they want to see.
+> The app will show the pokemon of that type. The use can see the name, type, and image of the pokemon. The user can also see the games the pokemon is in.
 
 
-## Features
 
-- Webpack setup 
-- HtmlWebpackPlugin
-- Loading CSS setup
-- Webpack-dev-server
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+## Built With
+
+- HTML, CSS, and Javascripts
+- Node.js
+- Web-hint, Stylelint
+- Boostrap
+- Jest
+- Webpack
+
+### Click link below for a live demo
+[Demo Link](https://github.com/Fourteen98/Pokemon/dist/)
```

Comment:
- [ ] The live demo link is broken, please fix it or remove it from your `README.md`  👌